src/components/calculator/calculator.css

```diff
@@ -0,0 +1,110 @@
+
+/* start index styles */
+
+/* end index styles */
+
+/* start universal styles */
+*,
+*::after,
+*::before {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Lato", sans-serif;
+  font-family: "Poppins", sans-serif;
+  font-size: 16px;
+  font-weight: 400px;
+  line-height: 1.42857143;
+  background-color: #191c23;
+  position: relative;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+::-webkit-scrollbar {
+  width: 6.5px;
+}
+
+::-webkit-scrollbar-track {
+  background-color: #e8dfec;
+}
+
+/* hsl(242 88% 56% / 0.8) */
+::-webkit-scrollbar-thumb {
+  height: 51.2413%;
+  background-color: #ff7b1c;
+  border-radius: 10px;
+}
+
+/* end universal styles */
+
+/* start common styles */
+ul {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+/* end common styles */
+
+/* start content styles */
+section {
+  height: 100vh;
+  display: grid;
+  place-items: center;
+}
+
+.sectionone__wrapper {
+  width: 400px;
+  box-shadow: 0 1px 12px azure;
+}
+
+.formcontrol > input {
+  width: 100%;
+  border: 0;
+  outline: 0;
+  padding: 29px;
+  text-align: right;
+}
```

Comment:
- [ ] As you are required to match the given design for this project, kindly change the input style from this:
>|![image](https://user-images.githubusercontent.com/91486791/205132270-ba5325af-18f1-40e6-82b1-d98a148426af.png)
to this: 
>
![image](https://user-images.githubusercontent.com/91486791/205132581-b867e3e6-746e-461a-b756-ca773b30ad67.png)

Reply:
- Thanks @tugsem  for reviewing my pull request I appreciate your review 💯 
### Thanks again for reviewing my PR  and have a good day😊

Reply:
issue is well resolved! ✔️